src/lexer.jl

```diff
@@ -312,7 +316,7 @@ function next_token(l::Lexer)
     elseif c == 'i'; return lex_i(l);
     elseif c == 't' || c == 'f'; return lex_bool(l);
```

Comment:
Ah ok, will a ` && VERSION >= v"0.6.0-dev.1471"`on line 965 of `lexer.jl`  fix it?